services/common/src/components/common/FileUpload.js

```diff
@@ -0,0 +1,280 @@
+/* eslint-disable */
+import React from "react";
```

Comment:
now that I merged my PR, I think it's just the Callout and LinkButton that aren't already in common.

Reply:
This is updated accordingly